roles/preflight-checks/tasks/ceph.yml

```diff
@@ -26,3 +26,8 @@
   delegate_to: "{{ item }}"
   with_items:
     - "{{ groups['controller'] }}"
+
+- name: fail if current ceph version is less than Jewel
```

Comment:
don;t we want to do this check on all the ceph-monitor and ceph-osd nodes? what if one of the ceph node is out of sync on version?